tools/about_paint.py

```diff
@@ -51,6 +52,9 @@ def traverse_paint(colr, callback):
 def f2dot14_safe(*values):
     return all(value >= -2.0 and value < 2.0 for value in values)
 
+def int16_safe(*values):
+    return all(-32768 <= value < 32768 and int(value) == round(value, 3) for value in values)
```

Comment:
in PR #319 (https://github.com/googlefonts/nanoemoji/pull/319/commits/8e80a190aa4e8870514645d1fe1f8309662196b9) you used `almost_equals(value, int(value))` instead of `== round(value, 3)`, should you do the same here?

Reply:
I think I'd go so far as to say we should have a shared function even